db/src/tx.rs

```diff
@@ -516,7 +520,7 @@ impl<'conn, 'a> Tx<'conn, 'a> {
     ///
     /// This approach is explained in https://github.com/mozilla/mentat/wiki/Transacting.
     // TODO: move this to the transactor layer.
-    pub fn transact_entities<I>(&mut self, entities: I) -> Result<TxReport> where I: IntoIterator<Item=Entity> {
+    pub fn transact_entities<I>(&mut self, entities: I) -> Result<(TxReport, AttributeSet)> where I: IntoIterator<Item=Entity> {
```

Comment:
I think it makes sense for the `AttributeSet` to be present inside the `TxReport` itself. Consider that Datomic actually provides the before and after **DBs**!